src/plugins/legacy/iterativeClosestPoint/iterativeClosestPointProcess.cpp

```diff
@@ -210,9 +247,19 @@ int iterativeClosestPointProcess::update()
     double* mat  = d->TransformMatrix->GetData();
     std::memcpy(data, mat, 16 * sizeof(*data));
 
-    vtkPolyData *output_polyData = ICPFilter->GetOutput();
+    vtkPolyData *output_polyData = TransformFilter2->GetOutput();
     output_polyData->GetFieldData()->AddArray(icpTransformArray);
 
+    // create array to hold RFE stats
+    vtkDoubleArray* rfeArray = vtkDoubleArray::New();
+    rfeArray->SetNumberOfComponents(3);
+    rfeArray->SetNumberOfTuples(1);
+    rfeArray->SetName("RFE Stats");
+    rfeArray->SetTuple3(0, d->mean, d->variance, d->median);
+    // add to outputs
+    output_polyData->GetFieldData()->AddArray(rfeArray);
+    rfeArray->Delete();
```

Comment:
You have a typo on "FRE" everywhere in ~toolbox.cpp and ~toolbox.h 

FRE = Fiducial Registration Error